lib/bitrise_conn.py

```diff
@@ -87,73 +31,63 @@ def get_apps(self):
     def set_app(self, project, apps):
         if apps is not None:
             if project == "android":
-                self.APP_SLUG = apps['data'][0]['slug']
+                self.BITRISE_APP_SLUG = apps['data'][0]['slug']
             elif project == "ios":
-                self.APP_SLUG = apps['data'][1]['slug']
+                self.BITRISE_APP_SLUG = apps['data'][1]['slug']
 
     def get_app(self, project, apps):
-        if not self.APP_SLUG:
-            resp = requests.get('https://api.bitrise.io/v0.1/apps/{0}'.
-                                format(self.APP_SLUG),
+        url = self.__url
+        if not self.BITRISE_APP_SLUG:
+            resp = requests.get('{0}''{1}'.
+                                format(url, self.BITRISE_APP_SLUG),
                                 headers=self.API_HEADER)
             if resp.status_code != 200:
                 raise _logger.error('GET /apps/ {}'.format(resp.status_code))
             return resp.json()
 
-    def get_workflows(self, APP_SLUG):
+    def get_workflows(self, BITRISE_APP_SLUG):
+        url = self.__url
         resp = \
-            requests.get('https://api.bitrise.io/v0.1/apps/{0}'
-                         '/build-workflows'.format(self.APP_SLUG),
+            requests.get('{0}{1}'
+                         '/build-workflows'.format(url, self.BITRISE_APP_SLUG),
                          headers=self.API_HEADER)
         if resp.status_code != 200:
             raise _logger.error('GET /apps/ {}'.format(resp.status_code))
         return resp.json()
 
-    def get_builds(self, params=None):
+    def get_builds(self, BITRISE_APP_SLUG, past_date_timestamp):
+        url = self.__url
+
+        # Change to BITRISE_HOST
         resp = \
-            requests.get('https://api.bitrise.io/v0.1/apps/{0}'
-                         '/builds'.format(self.APP_SLUG),
-                         headers=self.API_HEADER, params=params)
+            requests.get('{0}{1}'
+                         '/builds?after={2}'.format(url, BITRISE_APP_SLUG, past_date_timestamp), # noqa
+                         headers=self.API_HEADER)
         if resp.status_code != 200:
             raise _logger.error('GET /apps/ {}'.format(resp.status_code))
         return resp.json()
 
+    def get_builds_time(self, BITRISE_APP_SLUG, after):
```

Comment:
see above.

Also, is this a single time for multiple builds or is it meant to mean: "the build's time"?  If that's the case, I'd say instead:  `build_time()`